cli/QuickstepCli.cpp

```diff
@@ -150,6 +154,14 @@ DEFINE_bool(initialize_db, false, "If true, initialize a database.");
 DEFINE_bool(print_query, false,
             "Print each input query statement. This is useful when running a "
             "large number of queries in a batch.");
+DEFINE_string(profile_file_name, "",
```

Comment:
Alright. So, if I get it right, we need to provide three parameters, BUILD_TYPE as RelWithDebInfo, ENABLE_GOOGLE_PROFILER as ON and a profile_file_name string for getting the profiling output. There are two options, which if achievable, can make things a little bit simpler. 
1. When someone says ENABLE_GOOGLE_PROFILER is ON, we automatically change the BUILD_TYPE to RelWithDebInfo OR
2. Create a new BUILD_TYPE as "Profile", which internally takes care of enabling the profiling, ensuring that the necessary debug symbols are present in the code and optionally choosing a default profiling output filename, if one not specified. Once the quickstep process terminates, we can just print a message displaying the filename of the profiling output.